swagger.json

```diff
@@ -4508,6 +4624,27 @@
             }
           }
         },
+        "HosCertifiedShiftResponse": {
+            "type": "object",
+            "properties": {
+                "dayStartMs": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "Unix epoch time (in ms) of time of when this certified shift starts",
+                    "example": 1462881998034
+                },
+                "shippingDocIds": {
+                    "type": "string",
+                    "description": "Customer shipping document IDs, potentially multiple numbers delimited by a comma or space",
+                    "example": "1059984"
```

Comment:
nit: We may want the examples shown here match the examples within `hosCertifiedShiftParams`.